packages/pluggableWidgets/dropdown-sort-web/src/components/SortComponent.tsx

```diff
@@ -0,0 +1,98 @@
+import { createElement, ReactElement, useCallback, useEffect, useRef, useState } from "react";
+import { useOnClickOutside } from "@mendix/piw-utils-internal";
+import classNames from "classnames";
+
+export interface SortOption {
+    caption: string;
+    value: string;
+}
+
+interface SortComponentProps {
+    ariaLabel?: string;
+    defaultOption?: SortOption;
+    emptyOptionCaption?: string;
+    name?: string;
+    options: SortOption[];
+    tabIndex?: number;
+    updateSort?: (value: SortOption) => void;
+}
+
+export function SortComponent(props: SortComponentProps): ReactElement {
+    const [valueInput, setValueInput] = useState(props.defaultOption?.caption ?? props.emptyOptionCaption ?? "");
+    const [selectedSort, setSelectedSort] = useState<SortOption>(
+        props.defaultOption ?? {
+            caption: props.emptyOptionCaption ?? "",
+            value: ""
+        }
+    );
+    const [show, setShow] = useState(false);
+    const [dropdownWidth, setDropdownWidth] = useState(0);
+
+    const componentRef = useRef<HTMLDivElement>(null);
+
+    const onClick = useCallback((option: SortOption) => {
+        setValueInput(option.caption);
+        setSelectedSort(option);
+        setShow(false);
+    }, []);
+
+    useOnClickOutside(componentRef, () => setShow(false));
+
+    useEffect(() => {
+        if (selectedSort) {
+            props.updateSort?.(selectedSort);
+        }
+    }, [selectedSort]);
+
+    const showPlaceholder = !selectedSort || valueInput === props.emptyOptionCaption;
+
+    return (
+        <div className="dropdown-container" data-focusindex={props.tabIndex ?? 0} ref={componentRef}>
+            <input
+                value={!showPlaceholder ? valueInput : ""}
```

Comment:
Just do the following, makes it match with the below and avoids flipping conditionals
`value={showPlaceholder ? "" : valueInput}`

Reply:
Gonna fix :)

Reply:
Gonna fix

Reply:
Done